.gitignore

```diff
@@ -1,3 +1,36 @@
-assets/node_modules
-web/build
-assets/package-lock.json
+# OS Cruft
+.DS_Store
+
+# Package Managers
+vendor
+node_modules/
```

Comment:
Why does one of these have a trialing slash and the other does not?

Reply:
Fixed. Left by mistake.